src/org/labkey/test/tests/ExpTest.java

```diff
@@ -160,29 +159,25 @@ public void testSteps()
         selectQuery("exp", "Data"); // Select the one we want to edit
         waitForElement(Locator.linkWithText("edit metadata"), WAIT_FOR_JAVASCRIPT); //on Ext panel
         clickAndWait(Locator.linkWithText("edit metadata"));
-        waitForElement(Locator.xpath("//span[contains(text(), 'Reset to Default')]"), defaultWaitForPage);
-        int lastFieldIndex = getElementCount(Locator.xpath("//input[starts-with(@name, 'ff_label')]")) - 1;
-        Locator lastField = Locator.xpath("//input[@name='ff_label" + lastFieldIndex + "']");
-        click(lastField);
```

Comment:
Does the new designer just put the newly added alias field on the end of the list? If so, that matches this previous behavior. Looks like in the old GWT designer it was clicking the last field so that the alias field was added at the end.

Reply:
Yes, the alias field is added at the end of the list. Do we want to put the alias field next to the source field?

Reply:
Nope. I was just curious. I think that putting the alias field at the end makes the most sense. That is what the old GWT UI does after you save anyway, so putting it at the end right away sounds good to me.